doc/userguide/lua/libs/ja3.rst

```diff
@@ -0,0 +1,87 @@
+JA3
+---
+
+JA3 details are exposes to Lua scripts with the
+``suricata.ja3`` library, For example::
+
+  local ja3 = require("suricata.ja3")
+
+If you want to use ja3, you can either set suricata.yaml option
+``app-layer.protocols.tls.ja3-fingerprints`` to true,
+or specify it in the ``init`` function of your lua script
+by calling ``ja3.enable_ja3()``::
+
+  function init (args)
+    ja3.enable_ja3()
```

Comment:
how will this work if it only happens in a rule reload?

Reply:
Same way it happens now if you reload a new rule with `ja3.hash` or similar...

In the end, there is an atomic...

Looks ok, right ?

Reply:
ok great